src/libs/actions/Policy.js

```diff
@@ -22,7 +23,7 @@ Onyx.connect({
             return;
         }
 
-        allPolicies[key] = {...allPolicies[key], ...val};
+        allPolicies[key] = val;
```

Comment:
This was not allowing us to remove keys (like `errors`) from a policy. It kept adding it back.